dep_check/infra/go_parser.py

```diff
@@ -15,13 +16,34 @@ class GoParser(IParser):
     """
 
     def __init__(self):
+        self.lib_path = os.environ.get("GOLIB", f'{os.environ["HOME"]}/.dep-check')
+
         try:
-            self.lib = cdll.LoadLibrary("dep_check/lib/go_parse.so")
+            self.lib = cdll.LoadLibrary(f"{self.lib_path}/go_parse.so")
         except OSError:
-            logging.error(
-                "You have to build the go lib to parse go. Run 'make build-go'"
-            )
-            raise
+            logging.info("go_parse.so not found. Building...")
+            try:
+                os.system(f"go get -d github.com/lumapps/dep-check/dep_check/lib")
+                os.system(
+                    f"go build -o {self.lib_path}/go_parse.so -buildmode=c-shared "
+                    "$GOPATH/src/github.com/lumapps/dep-check/dep_check/lib/go_parse.go"
+                )
```

Comment:
prefer use `subprocess.run(..., check=True)` instead of `os.system` (cf. https://docs.python.org/3.7/library/subprocess.html - first paragraph)